modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-log-forwarding.adoc
+
+[id="cluster-logging-log-forwarding-about_{context}"]
+= About cluster log forwarding
+
+The {product-title} cluster log forwarding feature enables administrators to send collected log data 
+to external systems outside the {product-title} cluster. 
+
+[NOTE]
+====
+Enabling the log forwarding feature disables the xref:../../logging/config/cluster-logging-external.adoc#cluster-logging-external[default log forwarding feature].
+====
+
+Log forwarding enables you to use your existing logging service with your {product-title} cluster. You can send each type of log to different systems
+allowing you to control who in your organization can access each type. Optional TLS support ensures that you can send logs using secure communication
+as required by your organization.
+
+Log forwarding uses a combination of _outputs_ and _pipelines_ defined in the Log Forwarding Custom Resource. An output is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
+
+An output can be either:
+
+* `elasticsearch` to forward logs to an external Elasticsearch cluster, specified by server name or FQDN.
+* `forward` to forward logs to an external log aggregation solution. This option uses the Fluentd `forward` protocol.
+
+A log source is the type of data you can forward:
+
+* `logs.app` - Container logs generated by user applications running in the cluster, except infrastructure container applications.
+* `logs.infra` - Logs generated by both infrastructure components running in the cluster and {product-title} nodes, such as journal logs. Infra components are applications run on pods in the `openshift*`, `kube*`, or `default` projects.
+* `logs.audit` - Logs generated by the node Audit system (auditd), which are stored in the  */var/log/audit/audit.log* file, and the audit logs from the Kubernetes apiserver and the OpenShift apiserver. 
+
+Note the following:
+
+* Log forwarding does not provide secure storage for audit logs. You are responsible to ensure the endpoint is compliant with your organizational and governmental regulations and is properly secured. {product-title} cluster logging does not comply with those regulations.
+
+* You are responsible to create and maintain any additional configurations that external destinations might require, such as keys and secrets, service accounts, port opening, or global proxy configuration.
+
+* An output can support TLS communication using secrets. Secrets must have keys of: *tls.crt*, *tls.key*, and *ca-bundler.crt* which point to the respective certificates for which they represent. Secrets shall have the key *shared_key* for use when using forward in a secure manner.
+
+.Sample log forwarding outputs and pipelines
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
```

Comment:
Need to add name `instance` and that it is the only one supported, similar to the restriction with CL

Reply:
Outdated

Reply:
outdated